socorro/unittest/cron/jobs/test_bugzilla.py

```diff
@@ -377,3 +378,28 @@ def test_run_job_based_on_last_success(self):
         cursor.execute('select id, short_desc from bugs where id = 8')
         bug = cursor.fetchone()
         eq_(bug[1], 'newlines in sigs')
+
+    def test_find_signatures(self):
```

Comment:
This would be easy to rewrite using parametrize:

```
@pytest.mark.parametrize('content, expected', [
    # Simple signature
    ('[@ moz::signature]', set(['moz::signature'])),

    # 2 signatures and some junk
    ('@@3*&^!~[@ moz::signature][@   ns::old     ]', set(['moz::signature', 'ns::old'])),
    ...
])
def test_find_signatures(self, content, expected):
    assert find_signatures(content) == expected
```